app/src/main/java/org/schabi/newpipe/util/external_communication/TextLinkifier.java

```diff
@@ -33,7 +33,8 @@
 public final class TextLinkifier {
     public static final String TAG = TextLinkifier.class.getSimpleName();
 
-    private static final Pattern HASHTAGS_PATTERN = Pattern.compile("(#[A-Za-z0-9_]+)");
+    private static final Pattern HASHTAGS_PATTERN =
+            Pattern.compile("(#[A-Za-z0-9_\\u0080-\\uFFFF]+)");
```

Comment:
How does it handle URLs with an hashtag anchor like https://teamnewpipe.github.io/documentation/00_Prepare_everything/#content-that-is-not-permitted
If it didn't work beforehand then don't bother with it

Reply:
URLs with hashtag anchors work correctly. This does not change anything with this part of the code, and it was working before this commit. For this commit I tested this type of case and there was no change.

Reply:
Outdated